pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -200,6 +200,21 @@ type JaegerQuerySpec struct {
 
 	// +optional
 	JaegerCommonSpec `json:",inline,omitempty"`
+
+	// +optional
+    OauthProxy JaegerQueryOauthProxySpec `json:"oauthProxy,omitempty"`
```

Comment:
This can be a pointer, so that an empty object isn't persisted in case the user does not specify it.